### PR TITLE
Fix misc. compilation issues, update gitignore

### PR DESCRIPTION
**Pull request by ddcc**

### CODE REVIEW

**Comment by jpoirier** [comment]
The changes lgtm but deferring to the other _active_ devs.
-joe

File: src/rtl_ir.c [hunk 1]
```diff
@@ -77,7 +77,7 @@ void usage(void)
 		"\t[-b]\tDisplay output in binary (default), pulse=1, space=0; each 20 usec\n"
 		"\t[-t]\tDisplay output in text format\n"
 		"\t[-x]\tDisplay output in raw packed bytes, MSB=pulse/space, 7LSB=duration*20 usec\n"
-		"\t[-h\t]Help\n"
+        "\t[-h]\tHelp\n"
```
Comment (racerxdl) [inline src/rtl_ir.c]: Be carefull with the identation.

**Review by racerxdl — APPROVED** [review]
@ddcc the correct branch to make the PR is development (I already changed to development).

There is one conflict at src/librtlsdr.c (and a identation issue)

But in general is approved.

**Review by racerxdl — APPROVED** [review]
Looks ok to me

**Comment by racerxdl** [comment]
Ok so I got confused, this PR should be set to master since its small fixes for the master. Sorry. Its approved for me.

**Comment by jpoirier** [comment]
let him fix the indention then do the merge

**Comment by ddcc** [comment]
Updated

**Pull request merged**